pkg/config/config.go

```diff
@@ -372,7 +373,11 @@ func initConfig(config Config) {
 	// Note that trace-agent environment variables are parsed in pkg/trace/config/env.go
 	// since some of them require custom parsing algorithms. DO NOT add environment variable
 	// bindings here, add them there instead.
-	config.BindEnvAndSetDefault("apm_config.enabled", true)
+	if(runtime.GOARCH == "386") {
```

Comment:
Can you add a comment explaining why we added this conditional? It might be strange for someone reading this code.